Makefile

```diff
@@ -41,3 +41,6 @@ clean:
 		src/util/yp_strspn.c
 
 .PHONY: clean
+
+all-no-debug: DEBUG_FLAGS := -DNDEBUG=1
```

Comment:
I am definitely not super familiar with Makefiles - let me know if there's any odd syntax here....